governance/xc_admin/packages/crank_pythnet_relayer/src/index.ts

```diff
@@ -200,7 +203,7 @@ async function run() {
             .postInstructions([
               ComputeBudgetProgram.setComputeUnitLimit({ units: 1000000 }),
             ])
-            .rpc({ skipPreflight: true });
+            .rpc({ skipPreflight: false });
```

Comment:
I think we get better logs with simulation. Why was it true before?

Reply:
I normally got the logs from explorer, but it might be nicer to get them from Grafana